native/app/components/ListPanelHeader.js

```diff
@@ -10,15 +12,56 @@ class ListPanelHeader extends Component {
 
   constructor(props) {
     super(props);
+
+    this._resetSelection =  () => {
+      props.dispatch(resetSelect());
+    };
+
+    this._deleteSelection = () => {
+      const state = this.props.state;
+      const notes = state.notes.filter((note) => state.sync.selected.includes(note.id));
+
+      if (state.sync.selected) {
+        this.props.dispatch(deleteNotes(state.sync.selected, 'multi-delete'));
+      }
+      this.props.navigation.navigate('ListPanel', { deletedNote: notes });
+
+      // console.log(notes);
+      // props.dispatch(resetSelect());
+    }
   }
 
   render() {
     const { navigation } = this.props;
+    if (this.props.state.sync.selected) {
+      return (
+        <Toolbar style={styles.toolbarSelected}>
+          <ToolbarAction
+            size={20}
+            style={{ paddingTop: 4 }}
+            color='white'
+            icon='clear'
+            onPress={() => this._resetSelection()} />
+          <ToolbarContent
+            style={{ paddingLeft: 0,  }}
+            titleStyle={{ fontSize: 18, color: 'white' }}
+            title={ `${this.props.state.sync.selected.length} selected`} />
```

Comment:
This needs to be properly translated.

Reply:
Sadly, not really, adjectives are declined based on the number and need a proper plural form. For example, in Italian:
* 1 selected = 1 selezionato
* 2 selected = 2 selezionati

Reply:
@flodolo ok thank you!!

@sebastienbarbier we need a string that doesn't have numbers here

Reply:
Side question: are you going to use message.json to localize the add-on? It's not supported directly by Pontoon, so you'd need to go through properties -> JSON conversion.

Not sure if I'm looking at a mobile app or something else